editor/icons/Progress0.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path d="M9 1v3.055a4 4 0 0 1 1.027.426l2.159-2.159A7 7 0 0 0 9 1Z" style="fill:#e0e0e0;fill-opacity:.19607843"/><path fill-opacity=".196" d="M7 1.08A7 7 0 0 0 3.814 2.4l2.159 2.16A4 4 0 0 1 7 4.13V1.08ZM2.4 3.814A7 7 0 0 0 1.078 7h3.055a4 4 0 0 1 .426-1.027L2.4 3.814Zm11.2 0-2.16 2.159A4 4 0 0 1 11.87 7h3.05a7 7 0 0 0-1.32-3.186ZM1.08 9a7 7 0 0 0 1.32 3.186l2.158-2.159A4 4 0 0 1 4.13 9H1.08zm10.786 0a4 4 0 0 1-.425 1.027l2.158 2.159A7 7 0 0 0 14.92 9h-3.055zm-5.894 2.441L3.814 13.6a7 7 0 0 0 3.185 1.322v-3.055a4 4 0 0 1-1.027-.426Zm4.055 0a4 4 0 0 1-1.028.428v3.05a7 7 0 0 0 3.186-1.32l-2.158-2.158z" style="fill:#e0e0e0"/></svg>
```

Comment:
All the icons are missing their `viewBox` attribute, e.g. `viewBox="0 0 16 16"`. Also  `style="fill:#e0e0e0;fill-opacity:.19607843"` format is not supported by our color map convertor. These should be attributes. You can see this in the `Progress1.svg` icon.